src/frontend/src/store/index.ts

```diff
@@ -17,15 +21,44 @@ function uuidv4() {
   });
 }
 
+const genKeyPair = (): string[] => {
+  const keyPair = rs.KEYUTIL.generateKeypair("RSA", 1024);
+  console.log(keyPair)
+  return [rs.KEYUTIL.getPEM(keyPair.prvKeyObj, "PKCS1PRV"), rs.KEYUTIL.getPEM(keyPair.pubKeyObj)];
+}
+
+
+const keyPair = genKeyPair();
+console.log(keyPair);
+const privateKey = keyPair[0];
+const publicKey = keyPair[1];
+
+const getTransactionId = (transaction: Transaction): string => {
+  return sha256(
+    transaction.to + transaction.from + transaction.amount
+  ).toString();
+};
+
+const sign = (transaction: Transaction, privateKey: string): string => {
+  const dataToSign = transaction.id;
+  const sig = new rs.KJUR.crypto.Signature({alg: 'SHA256withRSA'})
+
+  sig.init(privateKey);
+  sig.updateString(dataToSign);
+
+  return sig.sign();
+};
+
 Vue.use(Vuex);
 Vue.use(VueToast);
 
 export default new Vuex.Store({
   state: {
     loading: false,
     walletCreated: false,
-    walletAmount: 0,
-    walletId: uuidv4(),
+    walletAmount: 50,
+    walletId: publicKey,//uuidv4(),
+    privKey: privateKey,
```

Comment:
Nitpick: Could you change privKey to privateKey for clarity please

Reply:
done